### DISCUSSION TITLE
Run test-runner against non-deployed Storybooks via Github Actions

### DISCUSSION THREAD

**Discussion by lucaslannes** [storybookjs/storybook#18833] [Help]
**Hi, everyone** 😄

I have installed Storybook to run automated UI tests via Github Actions, but I got a timeout running the test workflow.

![image](https://user-images.githubusercontent.com/54950765/182185819-6e88490c-d6ff-420a-8f90-2f3289dd165e.png)

I followed the instructions detailed in [Testing/Test runner](https://storybook.js.org/docs/react/writing-tests/test-runner) and copy/paste the workflow [Run against non-deployed Storybooks](https://storybook.js.org/docs/react/writing-tests/test-runner#run-against-non-deployed-storybooks) to my private repository.

So at this point I don't know if it is a hardware limitation of Github Actions or if I made something wrong.

The test-runner is working without errors on my local machine.

![image](https://user-images.githubusercontent.com/54950765/182187778-bb87f55a-25d3-4369-a1b2-8d8d91807622.png)

I'm working on a Next.js + React + Mantine project, using a Node.js 17.x version.

Maybe I need to migrate to Chromatic?

**Comment by franthormel** [comment]
Check this issue: https://github.com/storybookjs/test-runner/issues/301
